### PR TITLE
Update to include git repository source

### PR DESCRIPTION
**Pull request by johnsimons**
Part of https://github.com/OctopusDeploy/OctopusDeploy/pull/20337/

### CODE REVIEW

**Review by steve-fenton-octopus — COMMENTED** [review]
I've left a couple of Oxford commas for you in the suggestions.

**Comment by nikita-dergilev** [comment]
Do we need a section about how to configure sourcing from Git? Or is it self-explanatory?

**Comment by nikita-dergilev** [comment]
@johnsimons, could you please update this page too — https://octopus.com/docs/deployments/custom-scripts?

There is a section called `Where to store your scripts`

**Comment by nikita-dergilev** [comment]
@johnsimons, sorry, another comment. I didn't realise before but we don't have a page for the kubectl script step. Does it make sense to create one?

File: src/pages/docs/deployments/custom-scripts/aws-cli-scripts.md [hunk 1]
```diff
@@ -100,17 +100,25 @@ Get-STSCallerIdentity | Select-Object -Property *
...
-The first option is to add the script source code to the step directly. This is done by selecting the `Source code` option, and entering the source code into the text box in the `Script Content` section.
+The first option is to add the script source code to the step directly. This is done by selecting the `Inline script` option, and entering the source code into the text box in the `Inline Source Code` section.
 
 :::figure
 ![Source Code](/docs/deployments/custom-scripts/images/step-aws-script.png)
 :::
 
+### Git repository
+
+The second option is to run a script from a git repository. This is done by selecting the `Git repository` option, configuring the repository details, and entering the path name of the script file to run.
```
Comment (kirstenschwarzer) [inline src/pages/docs/deployments/custom-scripts/aws-cli-scripts.md:113]: Should this read 'entering the path to the script file you want to run'?

Comment (johnsimons) [reply src/pages/docs/deployments/custom-scripts/aws-cli-scripts.md]: Fixed

**Comment by johnsimons** [comment]
> could you please update this page too — [octopus.com/docs/deployments/custom-scripts](https://octopus.com/docs/deployments/custom-scripts)?
> 
> There is a section called `Where to store your scripts`

@nikita-dergilev I have added a new section but TBH that list is a bit messy!

**Comment by nikita-dergilev** [comment]
> > I didn't realise before but we don't have a page for the kubectl script step. Does it make sense to create one?
> 
> @nikita-dergilev I don't think we need it, I would actually condense all the permutation pages into one

@johnsimons , I agree, merging pages into one would be way better. Not sure if we have appetite for this now :-)

**Comment by nikita-dergilev** [comment]
> > could you please update this page too — [octopus.com/docs/deployments/custom-scripts](https://octopus.com/docs/deployments/custom-scripts)?
> > There is a section called `Where to store your scripts`
> 
> @nikita-dergilev I have added a new section but TBH that list is a bit messy!

Thanks, @johnsimons! I agree, it's messy :-)

**Review by scme0 — APPROVED** [review]
LGTM - we can merge the pages into one later.

**Comment by scme0** [comment]
⚠️ Note: We will merge this when the feature rolls out.

**Comment by scme0** [comment]
[sc-63146]

**Pull request merged**